package.json

```diff
@@ -28,9 +28,9 @@
   },
   "devDependencies": {
     "@babel/core": "^7.17.8",
-    "@skyscanner/eslint-config-skyscanner": "^12.0.0",
+    "@skyscanner/eslint-config-skyscanner": "^16.0.0",
```

Comment:
Updates loads of dependencies to latest.

Breaking here is the removal of babel-eslint, replaced internally

https://github.com/Skyscanner/eslint-config-skyscanner/releases